website/docs/advanced/user-management/saml/overview.md

```diff
@@ -0,0 +1,36 @@
+---
+id: sso-overview
+title: Overview
+---
+
+This section describes how you can enable SAML Single Sign-On (SSO) for your ConfigCat environment.
+
+SAML SSO allows your team members to authenticate with ConfigCat via their company accounts using your own Identity Provider (IdP).
```

Comment:
In the app I called this `SAML authentication` not `SAML SSO`. Any of them wins, it should be the same everywhere.

Reply:
✔️